### ISSUE TITLE
see my proj if it is possible

### ISSUE THREAD

**Issue by snip77** [Dionera/laravel-beanstalkd-admin-ui#16]
hello, i have a same of your project for redis manage.
see my project too if it is possible
[redis console](https://github.com/snip77/redis-console)

**Comment by ksassnowski** [comment]
I'm sorry, I don't understand what the question is. Can you rephrase it?

**Comment by spyp** [comment]
This package used for laravel but your project is php script

**Comment by ksassnowski** [comment]
I'm going to close this issue for now since I don't really understand what is being asked. Feel free to reopen it if you have more information.
